### PR TITLE
Prompt user for passwords in siac

### PR DESCRIPTION
**Pull request by triazo**
Also adds unconfirmed balance to status

### CODE REVIEW

**Comment by lukechampine** [comment]
looks good, although there's no escaping the fact that we're sending the password in plaintext over HTTP. We could switch to HTTPS, but that would only be delaying the inevitable.

File: siac/main.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func main() {
 
 	root.AddCommand(walletCmd)
 	walletCmd.AddCommand(walletAddressCmd, walletSendCmd, walletSiafundsCmd, walletStatusCmd, walletUnlockCmd, walletLockCmd, walletInitCmd, walletAddseedCmd, walletSeedsCmd)
-	walletInitCmd.Flags().StringVarP(&initPassword, "password", "p", "", "Encryption password used for wallet")
+	walletInitCmd.Flags().BoolVarP(&initPassword, "password", "p", false, "Encryption password used for wallet")
```
Comment (lukechampine) [inline siac/main.go]: need to change help text

**Comment by triazo** [comment]
I'll be updating documentation and including a readme in a seperate pull request on top of this.

File: siac/main.go (hunk 1 above)
Comment (DavidVorick) [inline siac/main.go]: you only need to supply a password in 3 commands (encrypt, unlock, recover). Encrypt I think by default should just use a blank password, and the others are required to use a password.

So the flag is probably not needed at all.

**Comment by DavidVorick** [comment]
I've merged this PR because I'm still uncertain about how the password situation should be handled. It might be more clear after some use.

**Pull request merged**